go/src/koding/socialapi/presence/presence.go

```diff
@@ -0,0 +1,38 @@
+package presence
+
+import (
+	"errors"
+	"net/http"
+	"net/url"
+
+	"koding/socialapi"
+)
+
+type Client struct {
```

Comment:
could you use the provided client in the socialapi/workers/presence/client here https://github.com/koding/koding/pull/9863/files#diff-9da7fe549db0224bc2de4a603ce84d51R47 (after you merge :D )